src/renderer/src/components/sideBar/treeFolder.vue

```diff
@@ -51,12 +51,14 @@
 </template>
 
 <script setup>
-import { ref, onMounted, nextTick } from 'vue'
+import { ref, onMounted, nextTick, computed } from 'vue'
 import { storeToRefs } from 'pinia'
 import { useProjectStore } from '@/store/project'
 import { showContextMenu } from '../../contextMenu/sideBar'
 import bus from '../../bus'
 import File from './treeFile.vue'
+// Import self for recursive rendering
+import Folder from './treeFolder.vue'
```

Comment:
I do not think you need to import itself? It is more of we need to define the component name as Folder if we want to still use Folder